### PR TITLE
Add note to installer that user can reuse robozonky-notifications.cfg file

### PR DESCRIPTION
**Pull request by liry**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -823,14 +823,4 @@
...
-  <repositories>
-    <repository>
-      <snapshots>
-        <enabled>false</enabled>
-      </snapshots>
-      <id>bintray-mockito-maven</id>
-      <name>bintray</name>
-      <url>https://dl.bintray.com/mockito/maven</url>
-    </repository>
-  </repositories>
```
Comment (triceo) [inline pom.xml]: Please revert this change. I would like to be able to use the latest Mockito, and for some reason, they don't publish their latest stuff to Central.

File: robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ protected AbstractStoryCondition(final Predicate<String> storyLength) {
     @Override
     public boolean test(final Wrapper<?> loan) {
         final String story = loan.getStory();
-        final boolean isStoryProvided = (story == null);
-        return storyLength.test(isStoryProvided ? "" : loan.getStory().trim());
+        final boolean isStoryNotProvided = (story == null);
+        return storyLength.test(isStoryNotProvided ? "" : loan.getStory().trim());
```
Comment (triceo) [inline robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java]: Good catch. 
Just to satisfy my curiosity: what were you looking for so deep in the code?

Comment (triceo) [reply robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java]: Also, if we're touching this, would you mind inverting the condition to `isStoryProvided`?
Will read better that way.

File: pom.xml (hunk 1 above)
Comment (liry) [reply pom.xml]: Well, this is quite sad situation. Maven is unable to download this dependency with my current settings.xml :(

Comment (liry) [reply pom.xml]: Ok, I'll revert it and try to find the root cause in my local settings.

File: robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java (hunk 2 above)
Comment (liry) [reply robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java]: Inverted.
FYI: I would like to add condition "story contains string ..."

File: pom.xml (hunk 1 above)
Comment (liry) [reply pom.xml]: Resolved. I had a `<mirror>` in my `settings.xml` with `<mirrorOf>*</mirrorOf>` ... had to change it to `<mirrorOf>!bintray-mockito-maven,*</mirrorOf>`

File: robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java (hunk 2 above)
Comment (triceo) [reply robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java]: OK, let me know if you need any help.

**Comment by triceo** [comment]
Thanks for the contribution!
